packages/dapp-toolkit/src/modules/wallet-request/request-resolver/request-resolver.module.ts

```diff
@@ -62,17 +62,19 @@ export const RequestResolverModule = (input: {
     requestItemModule.patch(interactionId, { sentToWallet: true })
 
   const addWalletResponses = (responses: WalletInteractionResponse[]) =>
-    ResultAsync.combine(responses.map(validateWalletResponse)).andThen(() =>
-      walletResponses.setItems(
-        responses.reduce<Record<string, WalletInteractionResponse>>(
-          (acc, response) => {
-            acc[response.interactionId] = response
-            return acc
-          },
-          {},
+    Result.combine(responses.map(validateWalletResponse))
+      .asyncAndThen(() =>
+        walletResponses.setItems(
+          responses.reduce<Record<string, WalletInteractionResponse>>(
+            (acc, response) => {
+              acc[response.interactionId] = response
+              return acc
+            },
+            {},
+          ),
         ),
-      ),
-    )
+      )
+      .mapErr((error) => logger?.error({ method: 'addWalletResponses', error }))
```

Comment:
returns the logger object `ILogObjMeta`

Reply:
Yeah saw that decided to not change as it's not a problem, this error is not consumed anywhere after that. I can make it return undefined

Reply:
Or map to `SdkError` just for the sake of consistency?

Reply:
It's not consumed. Does not really matter